### DISCUSSION TITLE
cn/2024/12/10/the-age-of-ai/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#195] [Announcements]
# cn/2024/12/10/the-age-of-ai/

https://hongtaoh.com/cn/2024/12/10/the-age-of-ai/

**Comment by silent-sure** [comment]
”……后面开始讲全球政治、冷战、军备竞赛，我就有点看不下去了……“
如果是我，估计看到作者是 Henry Kissinger 就不想看了。当年在智利策动政变的就是他。去年他去世的时候，Los Angeles Times 有一篇 Op-Ed，标题是 Saying goodbye to Kissinger the criminal。
